platform/android/app/src/main/kotlin/coop/polypoly/polypod/FeatureFragment.kt

```diff
@@ -112,15 +111,13 @@ open class FeatureFragment : Fragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        feature = FeatureStorage.featureForId(args.featureId)!!
```

Comment:
This might throw an exception if the id is incorrect, or categories have not been set. Where are we dealing with that exception?

Reply:
Right, this is a testing leftover.